app/javascript/components/pages/BlogShow.js

```diff
@@ -25,8 +33,9 @@ export class BlogShow extends Component {
         </NavLink>
         <br />
         <NavLink to="/blogindex" onClick={this.handleSubmit}>
-        <Button>Delete comment</Button>
-      </NavLink>
+          <Button>Delete comment</Button>
+        </NavLink>
+        {this.state.submitted && <Redirect to={`/blogindex`} />}
```

Comment:
`to={`/blogindex`}` does not need to be interpolated, since you are just passing a string.